src/components/table/Table.js

```diff
@@ -48,9 +48,26 @@ const FakeTableHeader = () => (
   </thead>
 );
 
+const getTableHeaderComponent = (arr, data) => {
+  if (data[0].type !== "line") {
+    // return basic header if the chart isn't a line chart
+    return <TableHeadersComponent arr={arr} />;
+  }
+  if (data.length === 1 || getIsIntervalData(data) !== true) {
+    // return basic header component if there's no interval data
+    // interval data is set to type scatter
+    return <TableHeadersComponent arr={arr} />;
+  }
+  // if the chart type is line and confidence intervals are displayed then return alternative header component
+  return <TableHeadersLineChartComponent arr={arr} data={data} />;
+};
+
+const getIsIntervalData = (series) => {
```

Comment:
we're defined this const twice - in javascipt-y land do you have the equivilent of helper functions that are shared across files? i.e `./helpers.js` or `./shared.js` or similar?

Reply:
I've moved it into `chart-helpers.js`